qa/suites/rbd/nvmeof/base/install.yaml

```diff
@@ -0,0 +1,36 @@
+use_shaman: True
+tasks:
+- cephadm:
+- cephadm.shell:
+    host.a:
+    # get state before nvmeof deployment
+    - ceph orch status
+    - ceph orch ps
+    - ceph orch ls
+    - ceph orch host ls
+    - ceph orch device ls
+    - ceph osd lspools
+    # create pool
+    - ceph osd pool create mypool
+    - rbd pool init -p mypool
+    - ceph osd pool ls detail
+    # deploy nvmeof
+    - ceph config set mgr mgr/cephadm/container_image_nvmeof quay.io/ceph/nvmeof:latest
```

Comment:
I'm curious about this line overriding the default image for nvmeof. It seems to me the test should, by default, test what is bundled in to the ceph build as that's what most users would get - since most won't be over-riding it this way. Is this line meant to be temporary?

Reply:
That was to test nvmeof images. I commented this line and added a note to uncomment it when testing specific nvmeof images.
cc: @caroav